### DISCUSSION TITLE
about data_id

### DISCUSSION THREAD

**Discussion by hecunjie** [infiniflow/ragflow#1450] [General]
how should i get data_id through new_conversation? I was wondering maybe I lost the user_id or other parameters in headers?

**Comment by KevinHuSh** [comment]
How about using postman?

**Comment by hecunjie** [comment]
I used postman but the retcode is 100



---Original---
From: "Kevin ***@***.***&gt;
Date: Wed, Jul 10, 2024 12:34 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [infiniflow/ragflow] about data_id (Discussion #1450)




 
How about using postman?

**Comment by KevinHuSh** [comment]
What about this setting:
![image](https://github.com/user-attachments/assets/58a8d0df-5a62-4fca-bd16-983bbfa9a819)
